components/transactions/TransactionDetails.js

```diff
@@ -179,14 +180,20 @@ const TransactionDetails = ({ displayActions, transaction, onMutationSuccess })
               <DetailDescription>{order.memo}</DetailDescription>
             </React.Fragment>
           )}
-          {order?.processedAt && (
+          {order?.processedAt && (transaction.kind === 'ADDED_FUNDS' || !paymentMethod) && (
```

Comment:
`!paymentMethod` will be true for any transaction without a payment method, that will include Expenses, etc  ...

Reply:
Makes sense. First I thought for expenses this isn't applied as this is only used within the `TransactionDetails.js` component and we don't show a details section for expenses. 

I've corrected this to specifically account only for contributions. 👍🏽